src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -41,27 +43,72 @@ const useStyles = makeStyles(() => ({
     transform: 'rotate(180deg)',
   },
   subMenu: {
-    backgroundColor: theme.palette.cruGrayDark.main,
+    backgroundImage: `linear-gradient(0deg, ${theme.palette.cruGrayDark.main}, ${theme.palette.cruGrayDark.main})`,
   },
   menuItem: {
     color: 'white',
     '&:hover': {
       backgroundColor: theme.palette.cruGrayMedium.main,
+      backgroundBlendMode: 'multiply',
     },
   },
   menuItemSelected: {
+    backgroundBlendMode: 'multiply',
     backgroundColor: theme.palette.cruGrayMedium.main,
   },
+  needsAttention: {
+    backgroundImage: `linear-gradient(0deg, ${theme.palette.mpdxYellow.main}, ${theme.palette.mpdxYellow.main})`,
+  },
   menuIcon: {
     marginRight: theme.spacing(1),
   },
+  notificationBox: {
+    backgroundColor: theme.palette.progressBarYellow.main,
+    paddingLeft: theme.spacing(1),
+    paddingRight: theme.spacing(1),
+    borderRadius: '25%',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginLeft: theme.spacing(2),
+  },
+  darkText: {
+    color: theme.palette.cruGrayDark.main,
+  },
+  whiteText: {
+    color: 'white',
+  },
 }));
 
+interface query {
+  totalCount: number;
+}
+
+interface responseData {
+  [key: string]: query;
+}
+
 const NavMenu = (): ReactElement => {
   const { t } = useTranslation();
   const classes = useStyles();
   const accountListId = useAccountListId();
   const currentToolId = useCurrentToolId();
+  const { data, loading } = useGetToolNotificationsQuery({
+    variables: { accountListId: accountListId ?? '' },
+  });
+  const [dataState, setDataState] = useState<responseData>({});
+  const [total, setTotal] = useState(0);
+
+  useEffect(() => {
+    const deepCopy = data ? JSON.parse(JSON.stringify(data)) : [];
+    setDataState(deepCopy);
+    setTotal(
+      Object.entries(deepCopy)
+        .map(([tool]) => deepCopy[tool])
+        .flatMap((entry) => entry.totalCount)
+        .reduce((a, b) => a + b, 0),
+    );
```

Comment:
I think `useMemo` would be better here than `useEffect` and `useState`. Honestly just doing the calculation every render probably wouldn't be the worst thing either. It probably isn't rerendered that much.